extensions/functions_rounding.yaml

```diff
@@ -3,21 +3,45 @@
 scalar_functions:
   -
     name: "ceil"
-    description: "Rounding to the ceiling of the value"
+    description: >
+      Rounding to the ceiling of the value `x`.
+
+      Note that the named option `rounding` refers to floating-point rounding
+      behavior and does not effect the behavior of `ceil` itself
     impls:
       - args:
           - value: fp32
+            name: "x"
+          - name: rounding
+            options: [ TIE_TO_EVEN, TIE_AWAY_FROM_ZERO, TRUNCATE, CEILING, FLOOR ]
+            required: false
```

Comment:
Ceil and floor can't ever round; the result can always be exactly represented because it necessarily is one of the function's inputs.